packages/web3-eth-contract/test/integration/contract_empty_string.test.ts

```diff
@@ -0,0 +1,50 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { Contract } from '../../src';
+import { getSystemTestProvider, createTempAccount } from '../fixtures/system_test_utils';
+import { MyContractAbi, MyContractBytecode } from '../fixtures/MyContract';
+
+describe('contract', () => {
+	describe('clone', () => {
```

Comment:
Maybe something more informative would be helpful.
The following is just a suggestion. And you may find something better :smile: 
```suggestion
describe('contract's function's calls', () => {
	describe('encoding edge cases', () => {
```